### DISCUSSION TITLE
[Question] OCaml related.

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#654] [Q&A]
![image](https://user-images.githubusercontent.com/89933766/224533534-25aa15b5-d8e0-4cf6-8076-23d63668b0d2.png)

What does " 'a " indicate in the highlighted part?

**Comment by m-spitfire** [comment]
It's a type variable, it means any type `'a`, it's kind of equivalent of generics in other languages, but type `'a` doesn't have any restrictions on it, it just means "any type", See [Polymorphic Functions](https://cs3110.github.io/textbook/chapters/basics/functions.html#polymorphic-functions)

**Comment by anisrashidov** [comment]
> It's a type variable, it means any type `'a`, it's kind of equivalent of generics in other languages, but type `'a` doesn't have any restrictions on it, it just means "any type", See [Polymorphic Functions](https://cs3110.github.io/textbook/chapters/basics/functions.html#polymorphic-functions)

Thank you
